libmamba/include/mamba/api/c_api.h

```diff
@@ -17,7 +18,10 @@ extern "C"
 {
 #endif
 
-    mamba::Configuration* mamba_new_configuration();
+    mamba::Context* mamba_new_context();
```

Comment:
Should expose the `ContextOptions` structure and have this function take it as an argument?

Reply:
Yes, I forgot to do that.

BTW did you check the other discussion with Hind where we consider the option to just remove the C API? Or do you know if it is actually used by anyone? I think it would be best provided as a separate library (at least a separate target)